build/FrameworkPackage/MakeFrameworkPackage.ps1

```diff
@@ -217,7 +226,16 @@ $compactxaml =
     </ResourceDictionary.MergedDictionaries>
 </ResourceDictionary>
 "@
-Set-Content -Value $compactxaml $fullOutputPath\Compact.xaml
+Set-Content -Value $compactxaml $DensityStylesDir\Compact.xaml
+
+# generate CustomizedPri
+$priConfigPath = [IO.Path]::GetFullPath("$CustomizedPriDir\priconfig.xml")
+$priOutputPath = [IO.Path]::GetFullPath("$CustomizedPriDir\CustomizedPri.pri")
+
+$makepriNew = "`"" + (Join-Path $WindowsSdkBinDir "makepri.exe") + "`" new /pr $CustomizedPriDir /cf $priConfigPath /of $priOutputPath /in $inputBaseFileName /o"
+Write-Host $makepriNew
```

Comment:
What does the .pri file look like here? I *think* the /in parameter specifies the namespace of the root, so that would put these resources in the Microsoft.UI.Xaml.2.1 namespace instead of the app.